src/environments/ui.ts

```diff
@@ -0,0 +1,462 @@
+/* eslint-disable */
+import { environment } from './environment';
+import { environmentTemplate } from './environment.template';
+import { Command } from 'commander';
+import * as fs from 'fs';
+import {EnclosureDispersalStrategy} from "../app/core/testing/enums/mock-storage.enum";
+const figlet = require("figlet");
+import { WebUiEnvironment } from "./environment.interface";
+
+interface CommandOptions {
+  [p: string]: any;
+}
+
+interface ReportOptions {
+  data?: unknown;
+  showHeader: boolean;
+  showFooter: boolean;
+}
+
+interface Headers {
+  header: string;
+  footer: string;
+}
+
+
+/*
+* Nice Header
+* */
+function showBanner(): void {
+  console.log(figlet.textSync('TrueNAS WebUI'));
+}
+
+function generateHeaders(content: string): Headers {
+  const width: number = 44;
+  const start = '\n ';
+  const finish = '\n';
+  const asterisk = '*';
+  const asterisksPerSide = (width - (content.length + 2)) / 2;
+  const asterisks = asterisk.repeat(asterisksPerSide);
+  const output = {
+    header: start +asterisks + ' ' + content.toUpperCase() + ' ' + asterisks + finish,
+    footer: start +asterisk.repeat(width) + '\n\n',
+  }
+
+  return output;
+}
+
+/*
+* Variables
+* */
+const environmentTs = './src/environments/environment.ts';
+const template = './src/environments/environment.template.ts';
+const originalEnvironment = {...environment};
+
+/*
+* Command Setup
+* */
+const program: Command = new Command();
+program
+  .version('0.0.1')
+  .description('WebUI setup utility')
+
+program
+  .command('mock')
```

Comment:
I feel like it may be easier if there was some sort of guided mode to this.
Right now, as a developer I don't know what models are available, which forces me to open IDE and look through files. At this point, I might as well edit environment.ts manually.
Maybe if it worked as a wizard, explaining every option, it would be easier.

Reply:
Done. You can try `./ui mock-gen` or with the alias `./ui mg`